scale/scale/settings.py

```diff
@@ -245,11 +248,18 @@
         'stream': sys.stdout
     },
     'console-err': {
```

Comment:
Don't we want to leave this at `WARNING` so we aren't duplicating messages between `stderr` and `stdout`?

Reply:
right now all logging messages go to stderr and info+debug go to stdout. all logging info should definitely show up on stderr, so you have one complete set of logs in a single stream when debugging. info_debug doesn't necessarily have to be on stdout.

Reply:
I definitely agree with using `stderr` for everything, but the goal of using `stdout` and `stderr` is for ease of delineation between the streams. Ultimately both streams are interleaved in the Scale log viewer, so we are just going to get duplicates of `INFO` and `DEBUG` levels as currently implemented. I think this should be revisited in the future, if we can support colorization of logs from a single stream based on level. I'd recommend putting in a UI issue for this.